frontend/src/styles.scss

```diff
@@ -12,3 +12,7 @@
         padding: 0 !important;
     }
 }
+
+.always-block {
```

Comment:
Overrides `hidden: true;` from Bootstrap. It only turns visible directly after an invalid control. I'm using it for form errors.